Libplanet/Crypto/IPrivateKey.cs

```diff
@@ -0,0 +1,39 @@
+using System.Collections.Generic;
+using Org.BouncyCastle.Crypto.Parameters;
+
+namespace Libplanet.Crypto
+{
+    // NOTE: Property KeyParam is ECPrivate/PublicKeyParameter. If a new algorithm digital signature
+    // is planned to be implemented, KeyParam should be interfaced or so.
+
+    /// <summary>
+    /// The interface of a private key. It allows to align the essential methods and
+    /// uses, mainly  <see cref="Sign"/> and <see cref="KeyBytes"/>.
+    /// </summary>
+    // ReSharper disable once InconsistentNaming
+    public interface IPrivateKey
+    {
+        /// <summary>
+        /// The <see cref="byte"/> representation of private key.
+        /// </summary>
+        IReadOnlyList<byte> KeyBytes { get; }
+
+        /// <summary>
+        /// The key parameter, <see cref="ECPrivateKeyParameters"/> of private key.
+        /// </summary>
+        ECPrivateKeyParameters KeyParam { get; }
```

Comment:
As discussed during daily, if this property can't be shared by implementations, I recommend removing it from the interface. 😗

Reply:
Amended at https://github.com/planetarium/libplanet/pull/2223/commits/95eeeb2124d08a54c32b03c490b5cc98723c29fe!